src/modules/languages/python.nix

```diff
@@ -582,7 +657,7 @@ in
     };
 
     enterShell = ''
-      export PYTHONPATH="$DEVENV_PROFILE/${package.sitePackages}''${PYTHONPATH:+:$PYTHONPATH}"
+      export PYTHONPATH="$DEVENV_PROFILE/${cfg.package.sitePackages}''${PYTHONPATH:+:$PYTHONPATH}"
```

Comment:
question: Why do we even need to set `PYTHONPATH`. We should never tinker with that if possible?

Reply:
ah weird: ok, so its an empty folder which gets filled by venv.enable I guess